src/components/radio-button/radio-button.spec.tsx

```diff
@@ -48,6 +50,21 @@ const borderColorsByValidationTypes = {
 };
 
 describe("RadioButton", () => {
+  let loggerSpy: jest.SpyInstance<void, [message: string]> | jest.Mock;
+
+  beforeEach(() => {
+    loggerSpy = jest.spyOn(Logger, "deprecate");
+    jest.restoreAllMocks();
+  });
+
```

Comment:
Doesn't appear to be a test added here, or any changes to the component file

Reply:
😶‍🌫️ Completely forgot about this component. Good spot.